cmd/coder/urls.go

```diff
@@ -84,26 +90,35 @@ func (sub createSubCmd) Run(fl *pflag.FlagSet) {
 	if !portIsValid(port) {
 		exitUsage(fl)
 	}
+	portNum, _ := strconv.Atoi(port)
```

Comment:
Ignoring err return here as port was just validated above. Can make portIsValid() return validated portNum instead if you'd prefer (probably would rename to validatePort() then)

```
portNum, err := validatePort(port) {
        if err != nil { exitUsage(f1) }
}
```